tests/test_assert_in_validators.py

```diff
@@ -21,15 +34,15 @@ def check_a(cls, v):
     with pytest.raises(ValidationError) as exc_info:
         Model(a='snap')
 
-    expected_errors = [
-        {
-            'type': 'assertion_error',
-            'loc': ('a',),
-            'msg': 'Assertion failed, invalid a',
-            'input': 'snap',
-            'ctx': {'error': 'invalid a'},
-        }
-    ]
-    actual_errors = exc_info.value.errors(include_url=False)
-    if expected_errors != actual_errors:
-        pytest.fail(f'Actual errors: {actual_errors}\nExpected errors: {expected_errors}')
+    _assert_eq(
```

Comment:
It looks like diff shows `+` for the right part but pytest uses `+` for the left part in `assert left == right`. Thus, this is reversed to match pytest.